### DISCUSSION TITLE
http/ssl

### DISCUSSION THREAD

**Discussion by jiacui** [isno/theByteBook#238] [General]
「非对称加密有两个密钥：公钥、私钥，私钥加密的密文只能公钥解，公钥加密的密文只能私钥解。」
——————————————————————————————
不完全准确，在非对称加密中，私钥加密通常指的是对数据进行签名，而不是传统意义上的加密。

1.5.3 服务网格 「如图 1-12 所示，服务网络由此得名」
——————————————————————————————
似乎应是「服务网格」

**Comment by isno** [comment]
**非对称加密** 算法（RSA）也能用于签名。但这里的主题是“加密”，所以我还是归类到“传统意义的加密”，只不过加密方形式不同。 这一块我理解也可能不对，如果错误的很严重再改。

第二个是笔误了，谢谢。
